src/puppetlabs/puppetdb/scf/storage.clj

```diff
@@ -1058,10 +1058,18 @@
                               (map #(vector certname-id %) new-package-ids)))
 
         (when (seq old-package-ids)
-          (jdbc/delete! :certname_packages
-                        ["certname_id = ? and package_id = ANY(?)"
-                         certname-id
-                         (sutils/array-to-param "bigint" Long (map long old-package-ids))]))))))
+          (let [old-package-id-sql-param (sutils/array-to-param "bigint" Long
+                                                                (map long old-package-ids))]
```

Comment:
Oh, I know where it's from then. You can get integers from SQL results, those need to flip to longs since we're declaring them to be longs.

👍